doc/arch/adr-006-geoprocessing-multioperation.md

```diff
@@ -0,0 +1,65 @@
+# Multi-Operation Geoprocessing Operation
+
+## Context
+
+### Current Operations
+
+Currently the geoprocessing service provides this set of operations:
+
+  * [`rasterGroupedCount`](https://github.com/WikiWatershed/mmw-geoprocessing/blob/c212f29e0b090dd9624b8f637e59fdf6cbbf4f4c/api/src/main/scala/Geoprocessing.scala#L240)
+  * [`rasterGroupedAverage`](https://github.com/WikiWatershed/mmw-geoprocessing/blob/c212f29e0b090dd9624b8f637e59fdf6cbbf4f4c/api/src/main/scala/Geoprocessing.scala#L192)
+  * [`rasterAverage`](https://github.com/WikiWatershed/mmw-geoprocessing/blob/c212f29e0b090dd9624b8f637e59fdf6cbbf4f4c/api/src/main/scala/Geoprocessing.scala#L149)
+  * [`rasterLinesJoin`](https://github.com/WikiWatershed/mmw-geoprocessing/blob/c212f29e0b090dd9624b8f637e59fdf6cbbf4f4c/api/src/main/scala/Geoprocessing.scala#L114)
+  * [`rasterSummary`](https://github.com/WikiWatershed/mmw-geoprocessing/blob/c212f29e0b090dd9624b8f637e59fdf6cbbf4f4c/api/src/main/scala/Geoprocessing.scala#L281)
+
+Each operation takes a multipolygon and a list of rasters, queries those rasters cropped to the multipolygon, and performs the respective calculation, and returns the result grouped by the given rasters.
```

Comment:
This may be a personal preference but consider adding line breaks at column 80 for each line.